commands/deps_resolve.go

```diff
@@ -70,6 +86,12 @@ var DepsResolveCommand = &cobra.Command{
 			}
 		}
 
+		if summaryTpl != nil {
+			if err := writeSummary(&report); err != nil {
+				logger.Log.Warnf("write summary file error: %v", err)
```

Comment:
What about

```suggestion
				return err
```

Reply:
I think it's better to perform before writing licenses if we need to throw the exception, it could make the command more consistent.